### PR TITLE
Add hdrp perf tests to the ruamel generator

### PR DESCRIPTION
**Pull request by theopnv**
### Purpose of this PR

Companion of #44 in the Private repository.
Merges the hdrp test perfs (for Windows) into the ruamel test generator.
Only the Memory test is working for now.

---
### Testing status
https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/yamato%252Fhd-perf-test

Perf tests: https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/yamato%252Fhd-perf-test/.yamato%252Fall-hdrp_performancetests.yml%2523Nightly_HDRP_PerformanceTests_trunk/4889709/job/pipeline

### CODE REVIEW

File: .yamato/all-hdrp_performancetests.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# Introduce any changes under .yamato/config/*.metafile files (for most cases) or under .yamato/ruamel/* within Python (more elaborate cases), and rerun build.py to regenerate all .yml files. 
+# Read more under .yamato/docs/readme.md 
+
+{% metadata_file .yamato/_latest_editor_versions_trunk.metafile -%}
+
+---
+
+PR_HDRP_PerformanceTests_trunk:
+    name: HDRP_PerformanceTests PR Job - trunk
+    dependencies: []
```
Comment (alelievr) [inline .yamato/all-hdrp_performancetests.yml:12]: Dependencies are empty for "all hdrp performance" config file, is this expected?

Comment (theo-pnv) [reply .yamato/all-hdrp_performancetests.yml:12]: I didn't add the tests part to any pipeline when starting this PR, I just focused on being able to run them manually. 
But good point, I added the memory test to the nightly in the last commit.
The Build editor one does not work, we'll have to investigate before adding it to the nightly.

Comment (theo-pnv) [reply .yamato/all-hdrp_performancetests.yml:12]: And I think we don't want to add it to the PR pipeline, only to the nightly. So it makes sense to keep the PR dependencies empty :)

Comment (alelievr) [reply .yamato/all-hdrp_performancetests.yml:12]: Ok, thanks :)

**Review by alelievr — APPROVED** [review]

**Pull request merged**